samples/web-nodejs-sample.devfile

```diff
@@ -0,0 +1,27 @@
+specVersion: 0.0.1
+name: nodejs-sample
+projects:
+  - name: nodejs-sample
+    source:
+      type: git
+      location: 'https://github.com/che-samples/web-nodejs-sample.git'
+tools:
+  - name: theia-editor
+    type: cheEditor
+    id: org.eclipse.che.editor.theia:1.0.0
+  - name: exec-plugin
+    type: chePlugin
+    id: che-machine-exec-plugin:0.0.1
+commands:
+  - name: start
+    actions:
+      - type: exec
+        tool: theia-editor
+        command: cd /projects/nodejs-sample/app && node app.js
+        workdir: /projects/nodejs-sample/app
+  - name: test
+    actions:
+      - type: exec
+        tool: theia-editor
+        command: curl ${HOSTNAME}:3000
```

Comment:
What is HOSTNAME here? Will be it automatically resolved inside a container?

Reply:
yes

Reply:
Could you elaborate more for my understanding? What is an example value in case of OS infrastructure? IP, service name, route host?

Reply:
This is probably a pod name, so we can access it in the same N/S

Reply:
Thanks for the explanation.
FYI
It's pod name
![Screenshot_20190311_121227](https://user-images.githubusercontent.com/5887312/54116227-fac8f300-43f6-11e9-8e69-144a47673bfb.png)
But it is not accessible outside of the pod, at least on my minikube installation
![Screenshot_20190311_121237](https://user-images.githubusercontent.com/5887312/54116259-0b796900-43f7-11e9-9afd-1fe418b032e0.png)

Reply:
let's change to `localhost` ?

Reply:
OK for me